index.html

```diff
@@ -1,17 +1,13 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-
-<head>
-  <meta charset="UTF-8" />
-  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>title</title>
-
-</head>
-
-<body>
-
-  <script type="module" src="/main.js"></script>
-</body>
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Загаловак</title>
```

Comment:
Грамматические ошибки исправить
```suggestion
    <title>Заголовок</title>
```

Reply:
Ок